compiler/noirc_evaluator/src/ssa/ssa_gen/program.rs

```diff
@@ -8,20 +8,16 @@ use serde_with::serde_as;
 use crate::ssa::ir::{
     function::{Function, FunctionId},
     map::AtomicCounter,
-    printer::display_instruction,
-    value::Value,
 };
 use noirc_frontend::hir_def::types::Type as HirType;
 
-use super::context::GlobalsContext;
-
 /// Contains the entire SSA representation of the program.
 #[serde_as]
 #[derive(Serialize, Deserialize)]
 pub(crate) struct Ssa {
     #[serde_as(as = "Vec<(_, _)>")]
     pub(crate) functions: BTreeMap<FunctionId, Function>,
-    pub(crate) globals: GlobalsContext,
+    pub(crate) globals: Function,
```

Comment:
Might still want a type alias or wrapper for this (can wait until a future PR).
That way users wouldn't have to know why this is a Function and we can limit that confusion/detail to its own module.